app/models.py

```diff
@@ -888,14 +888,14 @@ def apply(self, domain, post_records):
 
         # Adjustment to add multiple records which described in https://github.com/ngoduykhanh/PowerDNS-Admin/issues/5#issuecomment-181637576
         final_records = []
-        if NEW_SCHEMA:
-            records = sorted(records, key = lambda item: (item["name"], item["type"]))
-            for key, group in itertools.groupby(records, lambda item: (item["name"], item["type"])):
+        records = sorted(records, key = lambda item: (item["name"], item["type"], item["changetype"]))
+        for key, group in itertools.groupby(records, lambda item: (item["name"], item["type"], item["changetype"])):
```

Comment:
Only grouping by name+type (and changetype, but that shouldn't really matter since we set it statically in the previous part -- just felt it was The Right Way™ to not set it statically when we already did that).